packages/e2store/README.md

```diff
@@ -63,29 +66,29 @@ const e2hs = await formatE2HS(data)
 #### Read Tuples from E2HS
 
 ```ts
-import { readTuplesFromE2HS, parseEH2SBlockTuple } from "@ethereumjs/e2store"
-
-// Read all tuples from an E2HS file
-const tuples = await readTuplesFromE2HS(filePath)
-for await (const tuple of tuples) {
-  const { headerWithProof, body, receipts } = parseEH2SBlockTuple(tuple)
-  console.log(headerWithProof)
-  console.log(body)
-  console.log(receipts)
-}
+// ./examples/e2hs.ts#L29-L32
```

Comment:
Ah, we had this discussion on some of Amir's PRs already, so this by-line-inclusion tends to be really error-prone, since one is often not noticing if lines are shifting and this tends to get a constant re-hunting for correct documentation.

Then it's better to just keep static, will re-put the plain code here and push, branch needs an update anyhow.

Reply:
Update: ah, ok, I see the problem that the examples - if self-contained seem to get really long, also this makes use of this inclusion pretty heavily, a bit of a pity to reverse.

Then I won't do, but we should keep in mind a bit and be really careful here.

Reply:
Have pushed an update building the examples and do some small adjustments. Will approve/merge when CI passes. Thanks for the PR! 🙂